common/locales/en/messages.json

```diff
@@ -14,6 +15,7 @@
   "messageEquipped": " <%= itemText %> equipped.",
   "messageUnEquipped": "<%= itemText %> un-equipped.",
   "messageMissingEggPotion": "You're missing either that egg or that potion",
+  "messageInvalidEggPotionCombo": "Invalid egg+potion combination. Try hatching a different egg!",
```

Comment:
It'd only be quest eggs. The existing error message will still come up for trying to duplicate a pet combination. Thanks, I'll put that phrasing in!